_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1010,7 +1024,20 @@ Use these modules whenever possible. You should always use IAM roles in your own
 
 TODO(Rob): Add instructions for 1.19.
 
-TODO(Ina): Add instructions for 1.21.
+[[iam_access_analyzer_deployment]]
+=== Steps to enable IAM Access Analyzer for all enabled AWS regions
+To enable IAM Access Analyzer in your AWS Account or Organization, you need to do it separately for every region.
+
+To enable this for your AWS account:
+* you can do so manually using the AWS CLI or the AWS web console. For more guidance on how to do so, please refer to
+link:https://docs.aws.amazon.com/IAM/latest/UserGuide/what-is-access-analyzer.html[the official AWS docs];
+* you can also use the examples provided as part of `learning-and-testing/landingzone`:
```

Comment:
I think here in the deployment section we should specifically instruct to use our IaC library as opposed to manual configuration. So you can remove the bit that talks about manual enablement of this feature.